src/common/transformations/tests/op_conversions/sdpa_to_paged_attention_test.cpp

```diff
@@ -903,6 +903,7 @@ TEST_P(SDPAToPATest, SDPAToPA_Baichuan2_13b_General) {
     }
 }
 
+// clang-format off
```

Comment:
Why disable formatting?

Reply:
It's some unfortunate remaining - it broke my script with clang format check .. I'll remove it.